src/client/app/utils/input.ts

```diff
@@ -82,7 +82,9 @@ export const NoUnit: UnitData = {
 	suffix: '',
 	displayable: DisplayableType.none,
 	preferredDisplay: false,
-	note: ''
+	note: '',
+	minVal: -Infinity,
```

Comment:
Other places use ``adminSelectors`` to get the max/min allowed. I think it makes sense to do that here if possible. My only concern is this is outside a React component but it may work fine since not altering a value.